packages/amplify-cli/src/lib/init-steps/postInitSetup.js

```diff
@@ -0,0 +1,44 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const { getPackageManager } = require('../packageManagerHelpers')
+const { normalizePackageManagerForOS } = require('../packageManagerHelpers')
+
+async function run(context) {
+  if (context.parameters.options.app) {
+    // Pushing a sample app
+    try {
+      context.parameters.options.app = true;
+      context.parameters.options.y = true;
+      context.amplify.constructExeInfo(context);
+      await context.amplify.pushResources(context);
+      const packageManager = await getPackageManager();
+      await runPackage(packageManager);
```

Comment:
`runPackage()` has no arguments  is this the byproduct of a merge? Is the behaviour to take an argument or get packageManager within the function?

Reply:
getPackageManager should be inside the runPackage function only, removed it